baseplate/core.py

```diff
@@ -187,6 +190,7 @@ def configure_tracing(self, service_name,
                 num_span_workers=num_span_workers,
                 span_batch_interval=span_batch_interval,
                 sample_rate=sample_rate,
+                log_if_unconfigured=log_if_unconfigured,
```

Comment:
Yeah, I wonder if it should work like the statsd stuff where there's a function that standardizes config parsing and returns an object that can be passed to configure_tracing rather than having individual config options on configure_tracing. That lets the configuration parsing be standardized but still gives the application the ability to get at the raw object if it needs it (for e.g. non-server setups). 

```python
tracing_client = make_tracing_client(config)
...
baseplate.configure_tracing(tracing_client)
```